scripts/run_backend_tests.py

```diff
@@ -243,7 +245,7 @@ def _check_all_tasks(tasks):
             log(task_details)
 
 
-def _execute_tasks(tasks, batch_size=24):
+def _execute_tasks(tasks, batch_size=6):
```

Comment:
Dropping this to 12 earlier saved 10 mins, I'm dropping it to this 6 to see if it helps more. That being said, there probably isn't any more benefit to dropping it lower than this.

Reply:
There was a time when there was only one test running, I've increased this to 8.